tests/queries/0_stateless/01171_mv_select_insert_isolation_long.sh

```diff
@@ -42,14 +36,14 @@ function run_until_deadline_and_at_least_times()
     local function_to_run=$1; shift
 
     local started_time
-    started_time=$(get_now)
+    started_time=$SECONDS
     local i=0
 
     while true
     do
         $function_to_run $i "$@"
 
-        [[ $(get_now) -lt $deadline ]] || break
+        [[ $SECONDS -lt $deadline ]] || break
```

Comment:
400 seconds seem too much, no matter if in parallel or sequential. Why not break out if a certain  number of iterations is reached?

Reply:
> Why not break out if a certain number of iterations is reached?
It was like that recently. It required 100 iterations.
100 iteration on release take ~100 seconds. In debug it takes more that 10 minutes.
So we need to make this iteration limited by time.

I could add `MAX_ITERATION` that would be what you want.

Reply:
I see sub-10 second runs in master with release mode: https://s3.amazonaws.com/clickhouse-test-reports/0/8f60b9776fef6e14ca0e4b4a8b88ab329f17d3f2/stateless_tests__release_.html:
```
01171_mv_select_insert_isolation_long	OK	8.94
```

Reply:
In fact we can see that it has skyrocketed:

https://play.clickhouse.com/play?user=play#U2VsZWN0IHRvU3RhcnRPZkRheShjaGVja19zdGFydF90aW1lKSBhcyB0LCBhdmcodGVzdF9kdXJhdGlvbl9tcykKZnJvbSBjaGVja3MKd2hlcmUgdGVzdF9uYW1lID0gJzAxMTcxX212X3NlbGVjdF9pbnNlcnRfaXNvbGF0aW9uX2xvbmcnIGFuZCB0ZXN0X3N0YXR1cz0nT0snIEFORCBwdWxsX3JlcXVlc3RfbnVtYmVyID0gMCBhbmQgY2hlY2tfc3RhcnRfdGltZSA+ICcyMDI0LTA3LTAxIDAwOjAwOjAwJyBhbmQgY2hlY2tfbmFtZSA9ICdTdGF0ZWxlc3MgdGVzdHMgKHJlbGVhc2UpJwpHUk9VUCBieSB0Ck9SREVSIEJZIHQgQVND

It was 12 seconds until 10 of July, then it got 50%-100% slower around 10th of June, and now it's 20x slower

Reply:
yes, you are right, it is fast in release build

https://play.clickhouse.com/play?user=play#c2VsZWN0IGNoZWNrX25hbWUsIGNvdW50KCkgYXMgYywgcXVhbnRpbGVzKDAuNSwgMC45LCAwLjk5LCAwLjk5OTkpKHRlc3RfZHVyYXRpb25fbXMpLCBtYXgodGVzdF9kdXJhdGlvbl9tcykgYXMgbWF4X2QgZnJvbSBjaGVja3Mgd2hlcmUgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAgYW5kIGNoZWNrX3N0YXJ0X3RpbWUgPiAnMjAyNC0wMS0xMCAxODozMzoxMycgYW5kIHRlc3RfbmFtZSA9ICcwMTE3MV9tdl9zZWxlY3RfaW5zZXJ0X2lzb2xhdGlvbl9sb25nJyBhbmQgdGVzdF9zdGF0dXM9J09LJyBncm91cCBieSBjaGVja19uYW1lIGhhdmluZyBjID4gNTAgb3JkZXIgYnkgbWF4X2Q=